matlab/lib/+Ice/EndpointInfo.m

```diff
@@ -1,4 +1,4 @@
-classdef EndpointInfo < handle
```

Comment:
EndpointInfo and ConnectionInfo are immutable classes that don't need to be handles ... this was inherited from local Slice.